integration/build.gradle

```diff
@@ -29,7 +29,7 @@ dependencies {
         googleNotebook = "v1-rev20201110-1.30.10"
 
         datarepoClient = "1.0.155-SNAPSHOT"
```

Comment:
I would recommend that you also upgrade `kubernetesClient` above at line 18.

Or if you wish to wait and fix this issue as part of changing TestRunner's dependency into an app dependency, as @ddietterich suggested, that's fine with me.

You will need to create a JIRA ticket for this change and put the number in the PR title.

Reply:
Created Jira ticket [QA-1993](https://broadworkbench.atlassian.net/browse/QA-1993) as follow up.